PatientWebApplication/HealthClinic.DAL/Dtos/MedicineDescriptionDto.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace HealthClinic.CL.Dtos
+{
+    public class MedicineDescriptionDto
+    {
+        public String Name { get; set; }
+        public String Description { get; set; }
+        public int MedicineDescriptionId { get; set; }
+
+        public MedicineDescriptionDto() { }
+
+        public MedicineDescriptionDto(string name, String description, int medicineID)
```

Comment:
Change name of medicineID variable into medicineDescriptionId to match lines above